src/binder/binder_context.cpp

```diff
@@ -134,8 +185,26 @@ void BinderContext::SetColumnPosTuple(const std::string &col_name,
   expr->SetReturnValueType(column_object.Type());
 }
 
+void BinderContext::SetTableName(common::ManagedPointer<parser::ColumnValueExpression> expr,
+                                 common::ManagedPointer<parser::SelectStatement> node) {
+  if (node->GetSelectTable() != nullptr) {
```

Comment:
What if it is `nullptr`? Can that happen? Is it a failure scenario or a no-op? If `nullptr` means binding failure, we should throw. If `nullptr` can't happen, we should have an assertion. If the null check is just forcing a no-op, we should document the scenario when that happens.